docs/DEVELOPING.md

```diff
@@ -0,0 +1,192 @@
+# Developing LoopBack
+
+This document describes how to develop modules living in loopback-next monorepo.
+
+ - [Setting up development environment](#setting-up-development-environment)
+ - [Building the project](#building-the-project)
+ - [Running tests](#running-tests)
+ - [Coding rules](#coding-rules)
+ - [API documentation](#api-documentation)
+ - [Commit message guidelines](#commit-message-guidelines)
+ - [How to test infrastructure changes](#how-to-test-infrastructure-changes)
+
+## Setting up development environment
+
+Before you can start developing LoopBack, you need to install and configure few
+dependencies.
+
+ - [git](https://git-scm.com/): Github's [Set Up Git](https://help.github.com/articles/set-up-git/) guide is a good source of information.
+ - [Node.js 8.x (LTS)](https://nodejs.org/en/download/)
+
+You may want to configure your IDE or editor to get better support for
+TypeScript too.
+
+ - [VisualStudio Code](./VSCODE.md)
+ - _Missing your favorite IDE/editor here? Pull requests are welcome!_
```

Comment:
👍

Reply:
On the second thought, I am going to rewrite and expand the sentence `Pull requests are welcome!`, it has built negative connotations over the years.